juju_verify/verifiers/__init__.py

```diff
@@ -50,27 +50,28 @@ def get_verifier(units: List[Unit]) -> BaseVerifier:
     charm_types = set()
 
     if not units:
-        raise CharmException('List of units can not be empty when creating '
-                             'verifier')
+        raise CharmException("List of units can not be empty when creating " "verifier")
     for unit in units:
-        charm_type = parse_charm_name(unit.data.get('charm-url', ''))
-        logger.debug('Inferred charm for unit %s: %s', unit.entity_id,
-                     charm_type)
+        charm_type = parse_charm_name(unit.data.get("charm-url", ""))
+        logger.debug("Inferred charm for unit %s: %s", unit.entity_id, charm_type)
         charm_types.add(charm_type)
 
     if len(charm_types) > 1:
-        raise CharmException('Units are not running same charm. '
-                             'Detected types: {}'.format(charm_types))
+        raise CharmException(
+            "Units are not running same charm. "
+            "Detected types: {}".format(charm_types)
+        )
 
     charm = charm_types.pop()
 
     verifier = SUPPORTED_CHARMS.get(charm)
 
     if verifier is None:
         supported_charms = os.linesep.join(SUPPORTED_CHARMS.keys())
-        raise CharmException('Charm "{}" is not supported by juju-verify. '
-                             'Supported charms:{}'
-                             '{}'.format(charm, os.linesep, supported_charms))
-    logger.debug('Initiating verifier instance of class: %s',
-                 verifier.__name__)
+        raise CharmException(
+            'Charm "{}" is not supported by juju-verify. '
+            "Supported charms:{}"
+            "{}".format(charm, os.linesep, supported_charms)
```

Comment:
Do you know why it sometimes performs `"{}".format()` -> `f""` conversion and in this case it left the original `format()`?

Reply:
Yes, because I did couple myself.